### PR TITLE
feat: Badge on Avatar, Badge on Icons

### PR DESCRIPTION
**Pull request by Manishsharma203**
preview:
`https://codesandbox.io/s/musing-silence-1m1gns?file=/index.html`

### CODE REVIEW

File: Components/badges.css [hunk 1]
```diff
@@ -37,8 +37,23 @@
 .badge-success{
     background: var(--success);
 }
+.badge-avatar-online{
+    width: 10px;
```
Comment (athar05) [inline Components/badges.css:41]: I think rems can be used instead of px

File: docs.css [hunk 2]
```diff
@@ -75,6 +75,9 @@ a {
 .ml-auto{
   margin-left: auto;
 }
+.font-30{
+  font-size: 30px;
+}
 #themeToggle {
   font-size: 1.5rem;
```
Comment (athar05) [inline docs.css:82]: You have used two different units of measurement. I feel this can be avoided.

**Review by athar05 — COMMENTED** [review]
The UI looks good. I especially liked the vibrant look and feel of your CL.

File: docs.css (hunk 2 above)
Comment (Manishsharma203) [reply docs.css:82]: This is not a part of the CL only used for one specific use case.

File: Components/badges.css (hunk 1 above)
Comment (Manishsharma203) [reply Components/badges.css:41]: Only used here as width of this particular unit is required for this badge irrespective of browser/screen orientation

**Comment by Manishsharma203** [comment]
> The UI looks good. I especially liked the vibrant look and feel of your CL.

Thanks

> The UI looks good. I especially liked the vibrant look and feel of your CL.

Thanks Athar

**Pull request merged**